openerp/http.py

```diff
@@ -48,6 +49,10 @@
 from openerp.tools import ustr
 
 _logger = logging.getLogger(__name__)
+rpc_request = logging.getLogger(__name__ + '.rpc.request')
+rpc_response = logging.getLogger(__name__ + '.rpc.response')
+rpc_request_flag = rpc_request.isEnabledFor(logging.DEBUG)
+rpc_response_flag = rpc_response.isEnabledFor(logging.DEBUG)
```

Comment:
But the `rpc_*_flags` still need to be inside the dispatch(), as they can change at runtime ;-)